x-pack/test/cases_api_integration/spaces_only/tests/common/cases/patch_cases.ts

```diff
@@ -48,6 +48,13 @@ export default ({ getService }: FtrProviderContext): void => {
       expect(data).to.eql({
         ...postCaseResp(),
         title: 'new title',
+        customFields: [
```

Comment:
where does this come from? 😆  `postCaseReq` has none and we don't send `customFields` in the `updateCase` call.

Reply:
oops.. looks like I copied fix from `security_and_space` to here as well

Reply:
removed in [0c2de5e](https://github.com/elastic/kibana/pull/167472/commits/0c2de5efee9b3aac910ed74f3e464f91d3790ffa)